.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: required
+language: python
+services:
+  - docker
+python:
+  - "3.6.1"
```

Comment:
We should add a `2.7` line here too so tests run for both. Unless there's a reason I'm not remembering that Kubeface can't support 2.7